tests/FunctionalTestCase.php

```diff
@@ -571,6 +560,40 @@ protected function skipIfTransactionsAreNotSupported(): void
         }
     }
 
+    /**
+     * The Automatic Encryption Shared Library is installed with MongoDB Enterprise or Atlas (version 6.0 and later).
```

Comment:
This comment is going to be outdated once we start testing `crypt_shared` with older server versions going back to 4.2. See: [DRIVERS-2465](https://jira.mongodb.org/browse/DRIVERS-2465) and [DOCSP-29387](https://jira.mongodb.org/browse/DOCSP-29387) for more context.

I think both `@see` links are sufficient here and in `isMongocryptdAvailable()` below, if you'd like to remove the comment there as well.

Since I originally reported [PHPLIB-892](https://jira.mongodb.org/browse/PHPLIB-892), I've learned that MongoDB Enterprise (at least the server binary) is not relevant to whether the driver can perform automatic encryption, so I see no reason to mention it in a comment intended to be read by driver developers.